.rubocop_u2i.yml

```diff
@@ -0,0 +1,56 @@
+require: rubocop-rspec
+
+AllCops:
+  RunRailsCops: true
+  Include:
+    - '**/Rakefile'
+    - config.ru
```

Comment:
I know it's a copy-paste, but maybe it would make sense to get rid of unnecessary declarations?